core/src/streams.rs

```diff
@@ -697,22 +709,25 @@ impl<'gc> NetStream<'gc> {
         context: &mut UpdateContext<'_, 'gc>,
         variable_name: &[u8],
         variable_data: FlvValue,
-    ) {
+    ) -> Result<(), Avm2Error<'gc>> {
         match avm_object {
             Some(AvmObject::Avm1(object)) => {
                 match variable_name {
-                    b"onCuePoint" => {
+                    b"onCuePoint" | b"onMetaData" => {
```

Comment:
My assumption with FLV `ScriptData` was that it was a generic mechanism for calling methods on `NetStream`/`NetStream.client`. Any particular reason why there's a method name check here?